gateway/api_loader.go

```diff
@@ -220,17 +237,15 @@ func (gw *Gateway) processSpec(spec *APISpec, apisByListen map[string]int,
 
 	var prefix string
 	if spec.CustomMiddlewareBundle != "" {
-		if err := gw.loadBundle(spec); err != nil {
```

Comment:
removing loadBundle since we already load it during MakeSpec, we just need the dirver name here